ecs-cli/modules/commands/flags/flags.go

```diff
@@ -134,6 +134,7 @@ const (
 	RoleFlag                                = "role"
 	ComposeServiceTimeOutFlag               = "timeout"
 	ForceDeploymentFlag                     = "force-deployment"
+	LoadBalancerListFlag                    = "load-balancer-list"
```

Comment:
What do you think of make the flag name to be "target-groups", since this flag is to allow users to register their service to multiple target groups.

Reply:
Yes I think that makes more sense :)